### DISCUSSION TITLE
Example audio

### DISCUSSION THREAD

**Discussion by linuxplayground** [picocomputer/community#61] [Q&A] [answered]
Hi,

I ended up ordering my own PCBs from JLCPCB and the BOM from mouser and got it all soldered up yesterday and working on my Mac.  I can load and run the programs and VGA looks amazing.

But what about audio? I can't seem to find any examples of how to program the audio on the board.  I would like to know that it works at least.

I do see that the RIA firmware has code for audio and it looks like it has it is triggered on a timer to do the PWM stuff.  I just can't seem to find any examples or documentation on which registers to write to and what specifically to write to them to cause that function to generate any sound.

With a speaker plugged in, I can hear some faint feedback when programs are running.  Probably just a crappy audio cable and not intentional sound.

Am I missing something or is audio still on the TODO list?

Thanks!

**Comment by vruumllc** [comment, marked as answer]
Yeah, audio is still on TODO list. While there is some stuff on RIA, there is no connection to 6502 yet.  If you uncomment a line in aud.c in the RIA firmware and then build and install it, it will play a tone, as a test.

**Reply by linuxplayground** [reply]
ah cool. - I found it, built it and tested it.  Thanks.  Audio appears to work.  A beautifully annoying tone is produced.
